### DISCUSSION TITLE
很不错的驱动盘

### DISCUSSION THREAD

**Discussion by FYGT27** [share121/inter-knot#2327] [其他]
![Screenshot_2024-08-15-14-39-40-009_com miHoYo Nap](https://github.com/user-attachments/assets/c58e705d-92dc-45bb-bf5f-91c249956946)
想必咱的朱鸢有点怕痛啊

**Comment by MrDucker** [comment]
240分驱动盘，稳辣<img src="https://bbs-static.miyoushe.com/static/2024/06/19/ef939dd02fa4e48c6de1ca2ad661ac88_5558307353343114792.png" alt="萨姆 出击" width="75" >
